app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
+    </tr>
+    <tr>
+        <td><code>x-appwrite-id: [FILE-ID]</code></td>
+        <td>Required, contains which file this chunk belongs to.</td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-key: [API-KEY]</code></td>
+        <td>Optional, used for authentication in server integrations. <b>Do not use API keys in client applications.</b></td>
+    </tr>
+    </tbody>
+</table>
+
+<p>The multipart form data is structured as follows:</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 100px">Key</td>
+        <td style="width: 150px">Value</td>
+        <td style="width: 100px">File Name</td>
+        <td >Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>fileId</td>
+        <td><code>[FILE-ID]</code></td>
+        <td>n/a</td>
+        <td>Required, contains the file ID of the new file. Use <code>"unique()"</code> for a UUID.</td>
+    </tr>
+    <tr>
+        <td>file</td>
+        <td><code>[CHUNK-DATA]</code></td>
+        <td><code>[FILE-NAME]</code></td>
+        <td>Required, contains file chunk data.</td>
+    </tr>
+    <tr>
+        <td>permissions</td>
+        <td><code>[PERMISSION ARRAY]</code></td>
+        <td>n/a</td>
+        <td>Required, declares which file this chunk belongs to.</td>
+    </tr>
+    </tbody>
+</table>
+
+<p>While cURL and fetch are great tools to explore other REST endpoints, it's impractical to use for chunked file uploads because you need to split files into chunks.</p>
+
+<p>The multipart form data posted to file upload endpoints has the following format:</p>
+
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>POST /v1/storage/buckets/default/files  HTTP/1.1
+Host: [HOSTNAME_OR_IP]
+Content-Type: multipart/form-data; boundary=---WD9146A
+X-Appwrite-Project: demo-project
+content-range: bytes 10485760-12582912/12582912
+x-appwrite-id: 6369b0bc1dcf4ff59051
+
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="fileId"
+
+unique()
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="file"; filename="file.txt"
+Content-Type: application/octet-stream
+
+[CHUNKED-DATA]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="permissions"
+
+["read(\"user:627a958ded6424a98a9f\")"]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM--</code></pre>
+</div>
+
+<h2>Permissions</h2>
+
+<p>Appwrite SDKs have helpers to generate permission strings, but when using Appwrite without SDKs, you'd need to create the strings yourself.</p>
+
+<h3>Permission Types</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Permission String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Permission.read()	</code></td>
+        <td><code>"read(\"[PERMISSION_ROLE]\")"</code></td>
```

Comment:
All of these quotes and double quotes can be really confusing depending on the context 😕

Reply:
I thought this would make sense because it's a permission string. This is also just how it looks coming from the SDKs. Is a there a better way to format this?

Reply:
Would this be too confusing?

```suggestion
        <td><code>read("[PERMISSION_ROLE]")</code></td>
```

Either that or:

```suggestion
        <td><code>'read("[PERMISSION_ROLE]")'</code></td>
```